File: patch_locales.py
import json
with open('/var/www/html/locale/locales.json', 'r') as f:
    data = json.load(f)
data['Slovak'] = {
    'poEditor': 'sk',
    'locale': 'sk_SK',
    'languageCode': 'sk',
    'countryCode': 'SK',
    'nativeName': 'Slovenčina',
    'region': 'Europe',
    'dataTables': 'Slovak',
    'fullCalendar': True,
    'fullCalendarLocale': 'sk',
    'datePicker': True,
    'momentLocale': 'sk',
    'isRTL': False
}
with open('/var/www/html/locale/locales.json', 'w') as f:
    json.dump(data, f, ensure_ascii=False, indent=4)

File: patch_locales.php
<?php
$file = '/var/www/html/locale/locales.json';
$data = json_decode(file_get_contents($file), true);
$data['Slovak'] = [
    'poEditor' => 'sk',
    'locale' => 'sk_SK',
    'languageCode' => 'sk',
    'countryCode' => 'SK',
    'nativeName' => 'Slovenčina',
    'region' => 'Europe',
    'dataTables' => 'sk',
    'fullCalendar' => true,
    'fullCalendarLocale' => 'sk',
    'datePicker' => true,
    'momentLocale' => 'sk',
    'isRTL' => false,
];
file_put_contents($file, json_encode($data, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE));
